builder.go

```diff
@@ -44,7 +44,7 @@ ff02::3 ip6-allhosts
 	perm os.FileMode = 0644
 )
 
-var formats = []string{"qcow2", "qed", "raw", "vdi", "vhd", "vmdk"}
+var formats = []string{"qcow2", "qed", "raw", "vdi", "vhdx", "vhd", "vmdk"}
```

Comment:
`"vpc"` is missing, so if we run the command `d2vm convert -o alpine.vhd alpine`, the output will be converted from the `vhd` extension to `vpc` which will fail the validation.